pontoon/contributors/utils.py

```diff
@@ -104,6 +105,12 @@ def users_with_translations_counts(
     # Assign properties to user objects.
     contributors = User.objects.filter(pk__in=user_stats.keys())
```

Comment:
Instead of using the code block below, you can just add `profile__system_user=False` to the QuerySet.